src/user_templates_api/templates/jupyter_lab/templates/hra_api_tutorial/template.txt

```diff
@@ -0,0 +1,1283 @@
+[
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "40dd5488",
+    "metadata": {},
+    "source": [
+      "## HRA-API Documentation"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "id": "47b355bd",
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "# linked datasets\n",
+      "uuids = {{ uuids | safe }}"
+    ]
+  },
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "e2d89b95",
+    "metadata": {},
+    "source": [
+      "# INDEX\n",
+      "1. [Installation](#install) \n",
```

Comment:
I'm afraid these hyperlinks don't work in the workspace. Clicking on these doesn't change anything. Cmd+click (on mac) opens a new tab with the same workspace.

We can chat about this on slack

Reply:
Ok, I'll remove it in my next commit. It never seemed to work well with the linking.